content/projects/gsoc/index.adoc

```diff
@@ -41,11 +39,15 @@ We are applying to participate in Google Summer of Code 2023.
 // * link:/projects/gsoc/2022/projects/automatic-git-cache-maintenance[Automatic git cache maintenance on the controller] by Hrushikesh Rao
 // * link:/projects/gsoc/2022/projects/pipeline-step-documentation-generator[Pipeline Step Documentation Generator Improvements] by Vihaan Thora
 
-We are currently collecting project ideas. 
-Add your ideas by submitting an ad-hoc pull request as explained in our latest link:/blog/2022/11/16/gsoc-2023/[GSoC blog post].
+We are currently waiting to know whether we are accepted as a participating organisation (to be announced on Feb 22). 
```

Comment:
```suggestion
We are currently waiting to know wether we are accepted as a participating organisation (to be announced on Feb 22). 
```

Reply:
Thanks Stefan. :+1:

Reply:
Had a doubt on this and looked it up https://prowritingaid.com/wether-whether-weather